### ISSUE TITLE
Upgrade from C121 -> C122 on Win x64 via 1.64.x - Nightly (Checking C122)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#35907]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [ ] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by MadhaviSeelam** [comment]
Verification In progress using

```
Brave | 1.64.54 Chromium: 122.0.6261.29 (Official Build) nightly (64-bit)
-- | --
Revision | cf48cb2a6c35ed530efd5b15349ef4ee81301594
OS | Windows 11 Version 23H2 (Build 22631.3085)
```

<details>
<summary>C121 --> C122 on-demand upgrade check - Pending</summary>

* installed latest nightly `1.64.53 Chromium: 121.0.6167.139 ` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `   ` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|---------------|------------
![image](https://github.com/brave/brave-browser/assets/98358127/c109aeeb-c2ca-49c0-8a46-4adaaa079b62)|![image](https://github.com/brave/brave-browser/assets/98358127/f2a2f798-8dba-458e-8c5a-cfb42d7c7939)|![image](https://github.com/brave/brave-browser/assets/98358127/5baa56c6-4c9d-4974-8dfb-0ad885a9c391)

<details>
<summary>C120 --> C121 background upgrade check - Pending</summary>

* installed latest nightly `1.64.53 Chromium: 121.0.6167.139 and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{92BFBA70-A7D5-434D-9909-FE8301164450}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to ` ` 
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------|----------------


<details>
<summary>C121 --> C121 on-demand upgrade check - Pending</summary>

* installed `1.64.54 Chromium: 122.0.6261.29 ` via `BraveBrowserNightlySetup_122_1_64_54.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `       ` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|---------------|----------



<details>
<summary>C121 --> C121 background upgrade check - PASSED</summary>

* installed `1.64.54 Chromium: 122.0.6261.29 ` via `BraveBrowserNightlySetup_122_1_64_54.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{92BFBA70-A7D5-434D-9909-FE8301164450}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `    `
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------|------------

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x86` using the following build(s):

```
Brave | 1.64.57 Chromium: 122.0.6261.29 (Official Build) nightly (32-bit)
-- | --
Revision | 7d57398776cbf95dafe0c6544e0a64eb9f278379
OS | Windows 10 Version 22H2 (Build 19045.3996)
```

<details>
<summary>C121 --> C122 on-demand upgrade check - PASSED</summary>

* installed `1.64.53 Chromium: 121.0.6167.139` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.64.57 Chromium: 122.0.6261.29` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/2a853938-b956-4c1d-a37b-587e3abdc6ce) | ![image](https://github.com/brave/brave-browser/assets/2602313/459d815d-d4cb-493c-8b56-90ec1a5de3a6) | ![image](https://github.com/brave/brave-browser/assets/2602313/f1d35bae-7c62-4836-9dc4-d1006ae5c98b) | ![image](https://github.com/brave/brave-browser/assets/2602313/05b7b680-6f76-446c-897d-a50e2328e7d5)

</details>

<details>
<summary>C122 --> C122 on-demand upgrade check - PASSED</summary>

* installed `1.64.54 Chromium: 122.0.6261.29` via `BraveBrowserStandaloneNightlySetup32_122_1_64_54.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.64.57 Chromium: 122.0.6261.29` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/2e90077b-4daa-4b89-be6b-adc22d520c4f) | ![image](https://github.com/brave/brave-browser/assets/2602313/baa4a469-fb97-4b06-9607-35339559e0ae) | ![image](https://github.com/brave/brave-browser/assets/2602313/a48dd93e-302e-48c2-afab-688367b4c83a) | ![image](https://github.com/brave/brave-browser/assets/2602313/1699d85c-5e07-4f9a-864f-cf3e687aced7)

</details>

<details>
<summary>C121 --> C122 background upgrade check - PASSED</summary>

* installed `1.64.53 Chromium: 121.0.6167.139` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{B3C81AC2-A16A-4EEB-BB45-02A4E9209FCF}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.64.57 Chromium: 122.0.6261.29`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------|-----------------
![image](https://github.com/brave/brave-browser/assets/2602313/3208d8ea-991f-4943-b452-2d7198a805a6) | ![image](https://github.com/brave/brave-browser/assets/2602313/20607646-4471-49df-a2a4-ec5b0f75e3fe) | ![image](https://github.com/brave/brave-browser/assets/2602313/730c78f8-2c32-4a9a-8eb6-9695b3b5fa61) | ![image](https://github.com/brave/brave-browser/assets/2602313/4629bf27-8d1c-4c44-a0fc-0124ad682704)

</details>

<details>
<summary>C122 --> C122 background upgrade check - PASSED</summary>

* installed `1.64.54 Chromium: 122.0.6261.29` via `BraveBrowserStandaloneNightlySetup32_122_1_64_54.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{85643BAB-C99F-4BA2-81D4-4DD3D30A8437}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.64.57 Chromium: 122.0.6261.29`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/f3744313-3e4b-4533-b632-c3f140912604) | ![image](https://github.com/brave/brave-browser/assets/2602313/41191a2a-1a80-439b-8e3a-aec7f6aeadb8) | ![image](https://github.com/brave/brave-browser/assets/2602313/220ba6d4-7125-4edb-affb-692b3c84724c) | ![image](https://github.com/brave/brave-browser/assets/2602313/b34d2d01-0e0e-4391-ab34-ae25eb118e83)
</details>
